core/src/datayoga_core/blocks/http/receiver/block.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from abc import ABCMeta
+from asyncio import Queue
+from contextlib import suppress
+from itertools import count
+from typing import AsyncGenerator, List, Optional
+
+import orjson
+from aiohttp.web import BaseRequest, Response, Server, ServerRunner, TCPSite
+from datayoga_core.context import Context
+from datayoga_core.producer import Message
+from datayoga_core.producer import Producer as DyProducer
+
+logger = logging.getLogger("dy")
+
+
+class Block(DyProducer, metaclass=ABCMeta):
+    port: int
+
+    def init(self, context: Optional[Context] = None):
+        logger.debug(f"Initializing {self.get_block_name()}")
+        self.port = int(self.properties.get("encoding", 8080))
```

Comment:
better to take the default 8080 from the json schema itself so if we change it to 8081 we won't need to do it twice... This is relevant to other properties and their defaults.

Reply:
It's hard to do it here, because block don't have access to the schema. Thus, it must be refactored.

Reply:
OK, we probably should take care of it in a different task.